src/script/lua_api/l_mapgen.cpp

```diff
@@ -1606,6 +1617,111 @@ int ModApiMapgen::l_generate_decorations(lua_State *L)
 }
 
 
+// generate_biomes(vm, p1, p2, [noise_filler_depth])
+int ModApiMapgen::l_generate_biomes(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+
+	auto *emerge = getEmergeManager(L);
```

Comment:
unused?

Reply:
yea, dead code I forgot to remove, will fix that